src/Components/CurrentlyCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Card, Item } from 'semantic-ui-react';
+import CurrentItem from './CurrentItem';
+
+const CurrentlyCard = () => {
+  return (
+    <Card raised fluid>
+      <Card.Content>
+        <Card.Header>My current list</Card.Header>
+        <Item.Group divided style={{ maxHeight: '250px', overflow: 'auto' }}>
+          <CurrentItem
```

Comment:
`CurrentItem` sounds also like selected or presented in this moment.

Reply:
Same as above, naming should change to be more specific and self-explanatory as Item is already a predefined component from Semantic UI

Reply:
Yes, I think the same, only to be easier to distinguish them.